testplan/testing/multitest/driver/kafka.py

```diff
@@ -82,14 +82,24 @@ def __init__(
 
     @emphasized
     @property
-    def host(self) -> Optional[str]:
+    def host(self) -> str:
         """Host to bind to."""
+        if self.status != self.STATUS.STARTED:
```

Comment:
host and port might be used in config file instantiation from the template, I do not think we need to require these only available when started

Reply:
sure thing, maybe I've lost my mind when writing these lines 😵